File: dotnet/Logify.Alert.Core/Attachment.cs
using System.Collections.Generic;

namespace DevExpress.Logify.Core {
    public class Attachment {
        public string Name { get; set; }
        public string MimeType { get; set; }
        public byte[] Content { get; set; }
    }

    public class AttachmentCollection : List<Attachment> {
        public AttachmentCollection() {
        }
        AttachmentCollection(IEnumerable<Attachment> items) : base(items) {
        }
        internal AttachmentCollection Clone() {
            return new AttachmentCollection(this);
        }
    }
}

File: java/Core/src/main/java/com/devexpress/logify/alert/core/Attachment.java
package com.devexpress.logify.alert.core;

public class Attachment {
    private String name;
    private String mimeType;
    private byte[] content;

    public Attachment(String name, byte[] content) {
        this(name, "", content);
    }

    public Attachment(String name, String mimeType, byte[] content) {
        this.name = name;
        this.mimeType = mimeType;
        this.content = content;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMimeType() {
        return mimeType;
    }

    public void setMimeType(String mimeType) {
        this.mimeType = mimeType;
    }

    public byte[] getContent() {
        return content;
    }

    public void setContent(byte[] content) {
        this.content = content;
    }
}